### PR TITLE
Favorited by display

### PR DESCRIPTION
**Pull request by jonathanAgarcia**
Hey Team. Hoping to get review on my PR. I added a different display for favoritedBy amount (open to changes on the icon if anyone has ideas). Also added an api call for the friends search bar once clicked. The friends list will render the results of the api call.

### CODE REVIEW

File: front-end/src/components/CenterConsole/favoritedByDisplay.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import Box from '@material-ui/core/Box';
+import SentimentVeryDissatisfiedIcon from '@material-ui/icons/SentimentVeryDissatisfied';
+import SentimentDissatisfiedIcon from '@material-ui/icons/SentimentDissatisfied';
+import SentimentSatisfiedIcon from '@material-ui/icons/SentimentSatisfied';
+import SentimentSatisfiedAltIcon from '@material-ui/icons/SentimentSatisfiedAltOutlined';
+import SentimentVerySatisfiedIcon from '@material-ui/icons/SentimentVerySatisfied';
+
+const customFavIcons = {
+  1: {
+    icon: <SentimentSatisfiedIcon />,
```
Comment (rick-o-H) [inline front-end/src/components/CenterConsole/favoritedByDisplay.js:12]: this is awesome! Including sentiments is so cool

File: front-end/src/components/RightToolBar/RightToolBar.js [hunk 2]
```diff
@@ -1,21 +1,41 @@
...
   const handleFriendsSearch = (e) => {
     setUserInput(e.target.value);
+    if (e.target.value === '') {
+      setSearchResults([]);
+    }
   };
 
+  const searchBarClickHandler = async (e) => {
+
+    const fetchedUsers = await searchForFriends(userInput);
```
Comment (rick-o-H) [inline front-end/src/components/RightToolBar/RightToolBar.js:26]: Nice async await inside of a useEffect hook!!!

**Review by Doodlis — APPROVED** [review]
:smiley: :thumbsup:

File: front-end/src/utils/apiCalls.js [hunk 3]
```diff
@@ -13,6 +13,16 @@ export const postNewRecipe = (recipe, userId) => {
     });
 };
 
+export const searchForFriends = (userSearchTerm) => {
+  return axios.get(`http://localhost:3000/friends/${userSearchTerm}`)
+    .then((res) => {
+      return res;
+    })
+    .catch((err) =>{
+      return err;
```
Comment (rick-o-H) [inline front-end/src/utils/apiCalls.js:22]: Was going to say that returning the actual error could be problematic where you are calling this function, but you handled it beautifully with displaying data from state if status code is not 200!

**Review by rick-o-H — APPROVED** [review]
Looks great 🔥🔥🔥

File: front-end/src/utils/apiCalls.js (hunk 3 above)
Comment (jonathanAgarcia) [reply front-end/src/utils/apiCalls.js:22]: Thanks Rick, wasn’t sure about my error handling as well and am always open for suggestion

**Pull request merged**